src/css/footer.css

```diff
@@ -1,3 +1,145 @@
 .footer {
-  background-color: lightgreen;
+  background-color: var(--black);
+}
+
+.footer-main {
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  padding: 40px 0;
+}
+
+@media screen and (min-width: 768px) {
+  .footer-main {
+    flex-direction: row;
+    justify-content: space-between;
+    padding-top: 32px;
+    padding-bottom: 110px;
+  }
+}
+
+@media screen and (min-width: 1280px) {
+  .footer-main {
+    padding: 64px 0;
+  }
+}
+
+.footer-info .logo {
+  color: var(--white);
+}
+
+.footer-address {
+  font-style: normal;
+  padding-top: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .footer-address {
+    padding-top: 32px;
+  }
+}
+
+.contact-list {
+  flex-direction: column;
+  gap: 16px;
+}
+
+.contact-label,
+.footer-text,
+.policy-link {
+  line-height: 1.28571;
+  letter-spacing: -0.02em;
+  color: rgba(255, 255, 255, 0.6);
+  opacity: 0.9;
+}
+
+.contact-label {
+  display: block;
+}
+
+.contact-link,
+.footer-nav-link {
+  font-weight: 600;
+  line-height: 1.28571;
+  color: var(--white);
+  transition: color var(--transition);
+}
+
+.contact-link {
+  padding-top: 4px;
+}
+
+.contact-link:hover,
+.contact-link:focus,
+.footer-nav-link:hover,
+.footer-nav-link:focus {
+  color: var(--green);
+}
+
+.footer-nav-list {
+  flex-direction: column;
+  gap: 20px;
+}
+
+.footer-soc-list {
+  gap: 12px;
+}
+
+.footer-soc-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 42px;
+  height: 42px;
+  border: 1px solid rgba(255, 255, 255, 0.1);
+  border-radius: 10px;
+}
+
+.footer-soc-icon {
+  fill: var(--white);
+  opacity: 0.8;
+  transition: fill var(--transition);
+}
+
+.footer-soc-link:hover .footer-soc-icon,
+.footer-soc-link:focus .footer-soc-icon {
+  fill: var(--green);
+}
+
+.footer-hr {
+  border: 1px solid rgba(255, 255, 255, 0.1);

```

Comment:
Use border-top or border-bottom because with border you have both, but in design we have a line 1px